assets/spotify/artists-widget.js

```diff
@@ -0,0 +1,82 @@
+h.d;if(-1!==m.$||m.a)return b=Vb(b),-1===b.$?(c=b.e,p(ba,b.a,b.b,b.c,f(wa,a,b.d),c)):M}return p(u,c,d,e,f(wa,a,h),k)}m=td;b=7===Za.a?Za.f(a,b,c,d,e,h,k):Za(a)(b)(c)(d)(e)(h)(k);return f(m,a,b)}),td=e(function(a,b){if(-1===b.$){var c=b.a,d=b.b,e=b.c,h=b.d;b=b.e;if(ja(a,d)){a:for(a=b;;)if(-1===a.$&&-1===a.d.$)a=a.d;else break a;return-1===a.$?p(ba,c,a.b,a.c,h,$a(b)):M}return p(ba,c,d,e,h,f(wa,a,b))}return M}),Xb=e(function(a,b){a=f(wa,a,b);return-1!==a.$||a.a?a:p(u,1,a.b,a.c,a.d,a.e)}),rc=m(function(a,
+b,c){b=b(f(Tb,a,c));return b.$?f(Xb,a,c):q(Ub,a,b.a,c)}),ud=e(function(a,b){return q(Tc,"",Jb,f(Xa,b,a))}),Yb=e(function(a,b){return b.$?x(a(b.a)):{$:0,a:b.a}}),vd=function(a){return{$:4,a:a}},wd={$:2},xd={$:1},yd=e(function(a,b){switch(b.$){case 0:return x({$:0,a:b.a});case 1:return x(xd);case 2:return x(wd);case 3:return x({$:3,a:b.a.bx});default:return f(Yb,vd,a(b.b))}}),zd=e(function(a,b){return f(ud,a,yd(function(a){return f(Yb,Hb,f(Cc,b,a))}))}),Ad={$:0},Zb=e(function(a,b){return{aC:a,aM:b}}),
+Bd=y(f(Zb,Ma,r)),Cd=function(a){return H(function(b){var c=a.f;2===c.$&&c.c&&c.c();a.f=null;b(y(0))})},xa=m(function(a,b,c){for(;;)if(b.b){var d=b.a,e=b.b;if(d.$){var h=d.a;return f(S,function(b){var d=h.aO;return 1===d.$?q(xa,a,e,c):q(xa,a,e,q(Ub,d.a,b,c))},Ba(q(Sc,a,Na(a),h)))}var k=d.a;b=f(Tb,k,c);if(1===b.$)d=c,b=e,c=d;else return f(S,function(b){return q(xa,a,e,f(Xb,k,c))},Cd(b.a))}else return y(c)}),Dd=n(function(a,b,c,d){return f(S,function(a){return y(f(Zb,a,c))},q(xa,a,b,d.aC))}),Ed=m(function(a,
+b,c){a=a(b);return a.$?c:f(J,a.a,c)}),Fd=e(function(a,b){return q(Va,Ed(a),r,b)}),Gd=n(function(a,b,c,d){var e=d.b;return ja(b,d.a)?F(f(Na,a,e(c))):w}),Hd=m(function(a,b,c){return f(S,function(a){return y(c)},Qb(f(Fd,q(Gd,a,b.a,b.b),c.aM)))}),Id=e(function(a,b){return b.$?(b=b.a,{$:1,a:{aV:b.aV,aX:b.aX,a3:f(Uc,a,b.a3),al:b.al,bd:b.bd,bC:b.bC,aO:b.aO,bF:b.bF}}):{$:0,a:b.a}}),Jd=e(function(a,b){return{$:0,a:a,b:b}}),Kd=e(function(a,b){return f(Jd,b.a,f(Xa,b.b,a))});Y.Http={b:Bd,c:Dd,d:Hd,e:Id,f:Kd};
+var Ld=Fa("Http");Fa("Http");var Md=function(a){return Ld({$:1,a:{aV:!1,aX:Ad,a3:a.a3,al:r,bd:"GET",bC:w,aO:w,bF:a.bF}})}({a3:f(zd,f(Xa,function(a){return 1===a.$?{$:2,a:a.a}:{$:3,a:a.a}},function(a){return{$:1,a:a}}),sd),bF:"https://rodwrl4gq1.execute-api.us-east-1.amazonaws.com/Prod/hello/"}),Nd={$:2,m:r},$b=e(function(a,b){return a.$?G(a.a,Nd):G(od,Md)}),Od=f($b,{$:0},{$:0}),Pd={$:2,m:r},ya=e(function(a,b){return f(Hc,a,b)})("className"),T=ia("div"),Qd=e(function(a,b){return b.$?a:b.a}),Rd=function(a){var b=
```

Comment:
your AWS lambda url is "hello" :P
![image](https://user-images.githubusercontent.com/3020248/73407455-254dc400-42c7-11ea-978c-484bcea5424a.png)

Reply:
I'm curious, because I actually don't really know a lot about CORS - why isn't this CORS'd?

Reply:
It should be! I opened it up to the world for simplicity (yikes).
Also the `hello/` route was because I followed the tutorial, and haven't pushed a new one yet 😬 

I'm planning to do a technical hotfix push after this. Figured, not as much value in effort there until I release it to the world.

Reply:
ok cool... so it's the "access-control-allow-origin" configuration on the server that controls it I suppose